charts/portal/values.yaml

```diff
@@ -69,6 +69,7 @@ issuerComponentAddress: "https://ssi-credential-issuer.example.org"
 
 # -- Provide BPN DID Resolver Service base address
 bpnDidResolverAddress: "http://bpn-did-resolution-service-bdrs-server:8081"
+bpnDidResolverDirectoryAddress: "http://bpn-did-resolution-service-bdrs-server:8082"
```

Comment:
where is this values used?

Reply:
@Phil91 understood, would it be ok for you that I a push a commit to your feature branch to do some config improvements for the bpnDidResolver?

Reply:
@evegufy  sure :)